### PR TITLE
feat(WebexMember): display icon when member is sharing

### PR DESCRIPTION
**Pull request by lungancatalin**
Update WebexMember component to display content share icon
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-220730
https://www.screencast.com/t/haWhICjkBrO

### CODE REVIEW

File: src/data/memberships.js [hunk 1]
```diff
@@ -4,69 +4,69 @@ export default {
...
   },
   membership3: {
     ID: 'membership3',
     destinationID: 'meeting1',
     destinationType: 'meeting',
     members: [
-      {personID: 'user1', muted: false},
-      {personID: 'user2', muted: true},
-      {personID: 'user3', muted: true},
+      {personID: 'user1', muted: false, sharing: true},
```
Comment (maloun96) [inline src/data/memberships.js:28]: Only 1 user can share screen

File: src/data/memberships.js [hunk 2]
```diff
@@ -4,69 +4,69 @@ export default {
...
+      {personID: 'user2', muted: true, sharing: true},
+      {personID: 'user3', muted: true, sharing: false},
```
Comment (maloun96) [inline src/data/memberships.js:30]: Only 1 user can share screen

Comment (lungancatalin) [reply src/data/memberships.js:30]: Oh yeah, you're right! Thanks!

File: src/components/WebexMember/WebexMember.jsx [hunk 3]
```diff
@@ -27,12 +28,21 @@ export default function WebexMember({
...
-      {isMuted && <div className="member-muted"><span className="icon icon-microphone-muted_24" /></div>}
+      <div>
+        <div className="member-name">{`${firstName} ${lastName}`}</div>
+        <div className="member-presenter">{isSharing && <span>Presenter</span>}</div>
+      </div>
+      {isSharing && 
+          <div className="member-sharing"><Icon name="icon-content-share_16" /></div>
+      }
+      {isMuted && 
+        <div className="member-muted"><span className="icon icon-microphone-muted_24" /></div>
```
Comment (maloun96) [inline src/components/WebexMember/WebexMember.jsx:44]: Can you please update the icon to use `<Icon />` component as in the line `41` ?
```suggestion
        <div className="member-muted"><Icon name=" icon-microphone-muted_24" /></div>
```

File: src/components/WebexMember/WebexMember.scss [hunk 4]
```diff
@@ -12,9 +19,18 @@
...
   }
 
-  .member-muted {
+  .muted {
     flex: 1;
     text-align: right;
     color: #ff0000;
   }
+  .sharing {
+    // flex: auto;
```
Comment (maloun96) [inline src/components/WebexMember/WebexMember.scss:28]: remove comment

**Pull request closed without merging**